domains/transfer/Shared/DataContext/Migrations/20230515102933_InitialCreate.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using Microsoft.EntityFrameworkCore.Migrations;
 
-namespace API.Shared.Migrations;
+namespace DataContext.Migrations;
```

Comment:
Did you test that new migrations are added to this folder?

Reply:
Updated the readme in 20246f77 to support the new migration structure